options.go

```diff
@@ -26,7 +26,7 @@ type Options struct {
 	// httplog.SchemaGCP (Google Cloud Platform)
 	//
 	// Append .Concise(true) to reduce log verbosity, e.g. for localhost development.
-	Schema Schema
+	Schema *Schema
```

Comment:
This change is technically a "breaking API change". However, since we don't expect anyone to have used a custom schema yet and we're 2 days after initial v3 release, it's fine to merge.